src/viz-probation-population/VizProbationPopulation.js

```diff
@@ -0,0 +1,172 @@
+  font: ${(props) => props.theme.fonts.displayMedium};
+  font-size: 20px;
+  pointer-events: none;
+  text-anchor: middle;
+`;
+
+const OtherWrapper = styled.button`
+  background: none;
+  border: 0;
+  color: ${(props) =>
+    props.active ? props.theme.colors.highlight : props.theme.colors.heading};
+  cursor: pointer;
+  padding: 16px;
+  margin-left: -16px;
+  text-align: center;
+
+  &:hover {
+    color: ${(props) => props.theme.colors.highlight};
+  }
+`;
+
+const OtherLabel = styled.div`
+  font: ${(props) => props.theme.fonts.body};
+`;
+
+const findDistrictRecord = (district) => (record) =>
+  record.district === district;
+
+function ProbationMap({ currentLocation, data, onLocationClick, width }) {
+  // eslint-disable-next-line react/prop-types
+  const DistrictLabel = ({ hover, locationId, objectId }) => {
+    const districtRecord = data.find(findDistrictRecord(objectId));
+    return districtRecord ? (
+      <ParticipantCount
+        active={locationId === districtRecord.district}
+        hover={hover}
+      >
+        {districtRecord.value}
+      </ParticipantCount>
+    ) : null;
+  };
+
+  const otherRecord = data.find(findDistrictRecord(OTHER));
+
+  return (
+    <ProbationMapWrapper>
+      <StateMap
+        aspectRatio={ASPECT_RATIO}
+        LabelComponent={DistrictLabel}
+        locationId={currentLocation}
+        onRegionClick={onLocationClick}
+        stateTopology={judicialDistrictsTopology}
+        width={width}
+      />
+      {otherRecord && (
+        <OtherWrapper
+          active={currentLocation === OTHER}
+          onClick={(e) => {
+            e.preventDefault();
+            onLocationClick(OTHER);
+          }}
+        >
+          <ParticipantCount as="div">{otherRecord.value}</ParticipantCount>
+          <OtherLabel>{OTHER_LABEL}</OtherLabel>
+        </OtherWrapper>
+      )}
+    </ProbationMapWrapper>
+  );
+}
+
+ProbationMap.propTypes = {
+  currentLocation: PropTypes.string.isRequired,
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
+      district: PropTypes.string.isRequired,
+      value: PropTypes.number.isRequired,
+    })
+  ).isRequired,
+  onLocationClick: PropTypes.func.isRequired,
+  width: PropTypes.number,
+};
+
+ProbationMap.defaultProps = {
+  width: 0,
+};
+
+export default function VizProbationPopulation(props) {
+  const {
+    data: { populationDemographics, locations },
+    locationId,
+    onLocationClick,
+  } = props;
+
+  const judicialDistrictTotals = populationDemographics
+    .filter(recordIsTotal)
+    .map((record) => {
+      const judicialDistrictData = [...locations, { id: TOTAL_KEY }].find(
+        // these are stored as both strings and numbers;
+        // doing an extra typecast here just to be safe
+        (loc) => `${loc.id}` === `${record.district}`
+      );
+      if (judicialDistrictData) {
+        return {
+          district: `${record.district}`,
+          value: Number(record.total_supervision_count),
```

Comment:
For what it's worth, this `Number` casting approach is much easier to comprehend than the `+` approach 👍